### PR TITLE
Add timestamp check for scene_ctrl

### PR DESCRIPTION
**Pull request by Markushaldorsen**

### CODE REVIEW

File: adapter/service/scenectrl/routing_test.go [hunk 1]
```diff
@@ -22,6 +22,11 @@ const (
 	sceneUnsupported = "movietime"
 )
 
+var sceneReportColorloop = scenectrl.SceneReport{
```
Comment (boguslaw-wojcik) [inline adapter/service/scenectrl/routing_test.go:25]: It's used only in TestRouteSceneCtrl so you can probably move this var declaration there.

**Review by boguslaw-wojcik — COMMENTED** [review]
Cosmetic changes in tests?

File: adapter/service/scenectrl/routing_test.go (hunk 1 above)
Comment (Markushaldorsen) [reply adapter/service/scenectrl/routing_test.go]: Done

File: adapter/service/scenectrl/routing_test.go [hunk 2]
```diff
@@ -119,25 +124,38 @@ func TestRouteSceneCtrl(t *testing.T) { // nolint:paralleltest
 }
 
 func TestTaskSceneCtrl(t *testing.T) { // nolint:paralleltest
-	sceneColorLoop := "colorloop"
-	sceneNone := "none"
+	sceneReport1 := scenectrl.SceneReport{
+		Scene:     sceneColorloop,
+		Timestamp: time.Now().Add(-1 * time.Hour),
```
[the remark this answers is not in the document]
Comment (Markushaldorsen) [reply adapter/service/scenectrl/routing_test.go]: Done

**Review by boguslaw-wojcik — APPROVED** [review]

**Pull request merged**